src/pages/home/report/ReactionList/PopoverReactionList.js

```diff
@@ -74,46 +52,17 @@ class PopoverReactionList extends React.Component {
     }
 
     shouldComponentUpdate(nextProps, nextState) {
-        const selectedReaction = this.getSelectedReaction(nextProps.reportActions, nextState.reportActionID, nextState.emojiName);
-        const {emojiCount, emojiCodes, hasUserReacted, users} = this.getReactionInformation(selectedReaction);
         const previousLocale = lodashGet(this.props, 'preferredLocale', CONST.LOCALES.DEFAULT);
         const nextLocale = lodashGet(nextProps, 'preferredLocale', CONST.LOCALES.DEFAULT);
 
         return (
             this.state.isPopoverVisible !== nextState.isPopoverVisible ||
             this.state.popoverAnchorPosition !== nextState.popoverAnchorPosition ||
             previousLocale !== nextLocale ||
-            (this.state.isPopoverVisible &&
-                (this.state.reportActionID !== nextState.reportActionID ||
-                    this.state.emojiName !== nextState.emojiName ||
-                    this.state.emojiCount !== emojiCount ||
-                    this.state.hasUserReacted !== hasUserReacted ||
-                    !_.isEqual(this.state.emojiCodes, emojiCodes) ||
-                    !_.isEqual(this.state.users, users)))
+            (this.state.isPopoverVisible && (this.state.reportActionID !== nextState.reportActionID || this.state.emojiName !== nextState.emojiName))
```

Comment:
Why is it okay to remove all of these checks for count, hasReacted, etc? Are we not going to re-load more now?

Reply:
We are not passing emojiReactions via props. so no need to check for count, hasReacted, etc